extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -14,24 +14,16 @@
  * limitations under the License.
  */
 
-import {isJsonScriptTag, childElementsByTag} from '../../../src/dom';
+import {CSS} from '../../../build/amp-viz-vega-0.1.css';
+import * as dom from '../../../src/dom';
```

Comment:
confirmed its dce'ing correctly. only 2 module level functions i see are dom.childElementsByTag and dom.childElements (referenced by the former function). dom.isJsonScriptTag and dom.removeChildren don't show up as they are inlined on the call site.